src/dialect/postgresql.rs

```diff
@@ -230,15 +230,18 @@ pub fn parse_comment(parser: &mut Parser) -> Result<Statement, ParserError> {
 }
 
 pub fn parse_create(parser: &mut Parser) -> Option<Result<Statement, ParserError>> {
-    let name = parser.maybe_parse(|parser| -> Result<ObjectName, ParserError> {
+    if let Ok(name) = parser.maybe_parse(|parser| -> Result<ObjectName, ParserError> {
```

Comment:
Ah should this rather return the error? since the error would be a recursion error, so that the return type fot `parse_create` becomes `Result<Option<Result<statement`

Reply:
True, updated in the next commit